e2e/test/scenarios/organization/official-collections.cy.spec.js

```diff
@@ -145,6 +162,8 @@ function testOfficialBadgePresence(expectBadge = true) {
       name: "Official Dashboard",
       collection_id: collectionId,
     });
+
+    !expectBadge && setTokenFeatures("none");
```

Comment:
nit: I can understand why this has to be called here. But I feel like it's a little hard to read, since
https://github.com/metabase/metabase/blob/3b80635e56f0736eec1bd86b0f30d9f71d9ab8f6/e2e/test/scenarios/organization/official-collections.cy.spec.js#L139

basically sets the token features to have all features then it sets to `none` before going to the collection page. That's because if we set it to none in the first place we won't be able to create collection at all.

I think we might be able to structure this in a way that's easier to follow.